### ISSUE TITLE
URL to json api datasource

### ISSUE THREAD

**Issue by jkleinlercher** [cerndb/grafana-mimir-cardinality-dashboards#1]
although I get data when I curl my cardinality API like this

```
curl -X GET -H 'X-Scope-OrgID:anonymous' 'https://mimir.example.com/prometheus/api/v1/cardinality/label_names'
```

I get an error in grafana when I use the connection URL like in your readme

```
https://mimir.example.com/prometheus/api/v1/cardinality/
```

Error: `JSON API: Cannot connect to API`

Any idea? It works when I use the URL `https://mimir.example.com/prometheus/api/v1/cardinality/label_names` but then the dashboards don't work.

**Comment by BasJ93** [comment]
I tried it as well, and get the same error.

However, Grafana complains but still saves the URL for the datasource. Thus, set it to the shortened URL and save. Then load a dashboard and see that it works.

**Comment by kazeborja** [comment]
Indeed, even if Grafana complains when saving the URL the dashboards should still work as they add the missing bit in the plots, is it not the case for you?
